app/models/budget/investment/milestone.rb

```diff
@@ -0,0 +1,14 @@
+class Budget
+  class Investment
+    class Milestone < ActiveRecord::Base
+      belongs_to :investment
+
+      validates :title, presence: true
+      validates :investment, presence: true
+
+      def self.title_max_length
```

Comment:
Allright, talked about it, we'll catch up with this later on along with budget model usage of similar code https://github.com/consul/consul/blob/master/app/models/budget.rb#L34-L40